app/Config/Routes.php

```diff
@@ -20,4 +21,5 @@
 $routes->get('images/(:segment)', [Image::class, 'get']);
 $routes->get('events/(:num)', [Event::class, 'get']);
 $routes->get('events', [Event::class, 'get']);
+$routes->get('talks/(:num)/ics', [Talk::class, 'getICS']);
```

Comment:
This is pretty nice.
It would be nice when the `events` routs provides this with all other data.
Then I would not need to make an other API call for that. :)
`https://github.com/TechStreamConference/website-backend/blob/main/app/Config/Routes.php#L22`

Reply:
You wouldn't have to do another API call (if I understand it correctly). The provided route just serves as a download link. You you'd only have to put a link to this route in the frontend, you would not call the endpoint by yourself (it's similar to how you get the images from the backend).

Reply:
Ah okay.
Than Im fine with that.